React/examples/addPet.js

```diff
@@ -0,0 +1,19 @@
+// App.js
+addPet = (pet) => {
+  console.log('pet = ', pet);
+  axios.post('http://localhost:2999/pets', pet)
+    .then((response) => {
+      // We can update the state so we don't need to make another GET request
+      const updatedData = this.state.originalPets;
+      updatedData.push(response.data);
+      this.setState({
+        petList: updatedData,
+        originalPets: updatedData,
```

Comment:
It's awkward that we're updating `originalPets` since they're now no-longer the original ones.